templates/webinars/index.html

```diff
@@ -0,0 +1,112 @@
+                </div>
+      </div>
+   
+    <div class="u-fixed-width">
+      <hr style="margin: 2rem 0;">
+    </div>
+    
+        <div class="row u-equal-height">
+            <div class="row">
+                <div class="col-8">
+                  <h3><strong>Ubuntu Tech Live第2期：产品介绍</strong></h3>
+                </div>
+            </div>
+                <div class="col-6">
+                  <h2>Ubuntu Core 20新版本抢先新体验</h2>
+                  <p><strong>Taiten Peng &amp; Tony Chiang</strong></p>
+                  <p>Ubuntu Core 20最新特性和最新技术资讯介绍</p>
+                  <p><a href="/engage/ubuntu-tech-live-episode-2">注册观看</a></p>
+                </div>
+                <div class="col-6 u-hide--small u-align--center">      
+                  <img src="https://assets.ubuntu.com/v1/805f2b3a-core_black-orange_st_hex.svg" style="height: 240px;" alt="Ubuntu嵌入式" class="u-image-position--right">
+                </div>
+        </div>
+
+    <div class="u-fixed-width">
+      <hr style="margin: 2rem 0;">
+    </div>
+
+        <div class="row u-equal-height">
+            <div class="row">
+                <div class="col-8">
+                  <h3><strong>Ubuntu Tech Live第1期：产品介绍</strong></h3>
+                </div>
+            </div>
+                <div class="col-6">
+                  <h2>如何在Ubuntu上构建物联网项目</h2>
+                  <p><strong>Taiten Peng &amp; Tony Chiang</strong></p>
+                  <p>Canonical技术专家带来技术干活：在Ubuntu上快速构建高效和低成本的物联网项目</p>
+                  <p><a href="/engage/ubuntu-tech-live-episode-1">注册观看</a></p>
+            </div>
+                <div class="col-6 u-hide--small u-align--center">      
+                  <img src="https://assets.ubuntu.com/v1/805f2b3a-core_black-orange_st_hex.svg" style="height: 240px;" alt="Ubuntu嵌入式" class="u-image-position--right">
+                </div>
+        </div>
+    </section>
+
+    <section class="p-strip--light is-bordered">
+        <div class="row u-equal-height">
+            <div class="row">
+                <div class="col-8">
+                  <h3><strong>Ubuntu新系统发布活动</strong></h3>
+                </div>
+            </div>
+                <div class="col-6">
+                  <h2>Ubuntu 20.04线上发布活动回顾</h2>
+                  <p>在Ubuntu 20.04 LTS发布线上活动中有来自Ubuntu的技术专家和销售总监以及开源社区嘉宾，他们带来了最为全面的新版本介绍，Ubuntu在云端、物联网设备端相关产品介绍以及如何构建开源社区等内容。这不仅是一次新版本发布派对，还是一次技术分享干货盛会。</p>
+                  <p><a href="https://cn.ubuntu.com/blog/ubuntu-20-04-lts-launch-event">前往观看</a></p>
+                </div>
+                <div class="col-6 u-hide--small u-align--center">      
+                  <img src="https://assets.ubuntu.com/v1/e72b40fa-focal-default-background.png" style="height: 240px;" alt="Ubuntu-20.04" class="u-image-position--right">
+                </div>
+        </div>
+    </section>
+
+
+
+
+
+
+
+</div>
+
+
+
+
+{% endblock %}
```

Comment:
@zprood Can you remove all the additional space and end the file with a new line please? Thank you.

Reply:
@carkod Yes, done. Thank you.